pysat/utils/testing.py

```diff
@@ -152,24 +157,82 @@ def eval_warnings(warns, check_msgs, warn_type=DeprecationWarning):
     return
 
 
-def prep_dir(data_path):
-    """Prepare the directory to provide netCDF export file support.
+def create_files(inst, start, stop, freq=None, use_doy=True, root_fname=None,
```

Comment:
Since this function is clearly tied to the testing Instruments, would it make more sense for this to live in `pysat.instruments.methods.testing`?  Open to either location.

Reply:
Moved to `methods.testing`